README.md

```diff
@@ -113,6 +118,11 @@ You can also open an issue for a feature request.
 
 If you find an issue with this package and have a fix, please feel free to open a pull request following the [procedures](https://github.com/DataDog/datadog-cdk-constructs/blob/main/CONTRIBUTING.md).
 
+## Testing
+
+If you contribute to this package you can run the tests using `yarn test`. This package also includes a sample application for manual testing. First open a seperate terminal and run `yarn watch`, this will ensure the Typescript files in the src directory are compiled to Javascript in the lib directory. Navigate to `src/sample`, here you can edit `default.integration.ts` to your liking and test your contributions manually. Run `npx cdk --app lib/sample/default.integration.js <CDK Command>`, replacing `<CDK Command>` with common CDK commands like `synth`, `diff`, or `deploy`. 
+* Note, if you recieve "... is not authorized to preform: ..." you may also need to provide your AWS profile through AWS-Vault or any similar program.
```

Comment:
1. preform -> perform
2. I wouldn't mention aws-vault, that's something we use here at Datadog but may not be commonly used by anyone who is contributing to our library.

Reply:
Regarding (2), I rephrased it to "Note, if you receive "... is not authorized to preform: ..." you may also need to authorize the commands with your AWS credentials" Im not sure if that is the best way to say it.